convince-my-boss.html

```diff
@@ -126,22 +138,31 @@ <h2 class="small-caps center">Convince My Boss</h2>
             </li>
           </ul>
         </p>
-        <p>It is
-          <strong>THE</strong> conference to go to for Python professionals. You can learn more about the conference and last year's
-          talk and sessions
-          <a href="https://in.pycon.org">
-            <strong>here</strong>
-          </a>
-        </p>
+        <p>It is <strong>THE</strong> conference to go to for Python professionals. You can learn more about the conference and last year's
+          talk and sessions <a href="https://in.pycon.org"><strong>here</strong></a>.</p>
         <p>If I register before 30th of June, the fee is only 1200 rupees. Included in the conference ticket is access to all
           PyCon India keynotes and talks, sponsor showcase, community spaces (if included this time) and many other networking
           events, an official PyCon India T-shirt along with lunch and high-tea in the evening for both days (6th-7th October)</p>
         <p>I would be happy to write a post-conference report detailing what I’ve learned and to recommend any changes that
           we could implement to improve our practices. Please let me know if you have any questions or concerns. I hope we
           can make this opportunity work.</p>
-
         <p>Thank you,
-          <br> {yourName}</p>
+        <br> {yourName}</p>
+      </div>
+      <div id="download" class="d-block text-center">
+        <a href="convince-my-boss.docx" class="button btn-download" download="convince-my-boss.docx" id="magic" onclick="happy()">
+          <i class="fa fa-cloud-download icon"></i>Download Document</a>
+        <a href="convince-my-boss.txt" class="button btn-download" download="convince-my-boss.txt" id="magic1" onclick="happy1()">
+          <i class="fa fa-file icon"></i>Download Plain-text</a>
+
+        <script>
+          function happy() {
+            document.getElementById("magic").innerHTML = "Happy Convincing !!";
```

Comment:
Uh the button actually looks better without this effect.

Reply:
Removing then..